src/main/java/com/github/checkstyle/regression/Main.java

```diff
@@ -19,11 +19,45 @@
 
 package com.github.checkstyle.regression;
 
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+import com.github.checkstyle.regression.configuration.ConfigGenerator;
+import com.github.checkstyle.regression.data.GitChange;
+import com.github.checkstyle.regression.data.ModuleExtractInfo;
+import com.github.checkstyle.regression.data.ModuleInfo;
+import com.github.checkstyle.regression.extract.ExtractInfoProcessor;
+import com.github.checkstyle.regression.git.DiffParser;
+import com.github.checkstyle.regression.module.ModuleCollector;
+import com.github.checkstyle.regression.module.ModuleUtils;
+import com.github.checkstyle.regression.report.ReportGenerator;
+
 /**
  * Utility class, contains main function and its auxiliary routines.
  * @author LuoLiangchen
  */
 public final class Main {
+    /** Option name of repository path. */
+    private static final String OPT_REPO = "repo";
```

Comment:
Add short names and long names.

https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/diff.groovy#L44
Try to make similar options the same as the groovy scripts, that way there is less a learning curve.
This sound good?